pkg/test/resourcefixture/testdata/basic/bigquery/v1beta1/bigquerydatasetaccessblock/_http.log

```diff
@@ -0,0 +1,421 @@
+POST https://bigquery.googleapis.com/bigquery/v2/projects/${projectId}/datasets?alt=json
```

Comment:
Remove?

Reply:
Good call, will move to the mockgcp implementation for BigQuery